### PR TITLE
feat: Explain how to verify kwctl binaries

### LINKED ISSUE #229: Instructions about how to verify kwctl binary
write instructions about how to verify the binary downloaded using the sigstore signatures

### PR DESCRIPTION
**Pull request by raulcabello**
Fix #229

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -282,3 +282,21 @@ rm ~/.zcompdump*
 ```
 
 Then start a new shell or run `source ~/.zshrc` once.
+
+## Verify kwctl binaries
+
+kwctl binaries are signed using [Sigstore](https://docs.sigstore.dev/cosign/working_with_blobs/#signing-blobs-as-files). 
+Each release contains two files: kwctl.sig and kwctl.pem that can be used for verification.
```
Comment (flavio) [inline README.md:289]: How do I download these files and make sure they are not tampered?

Can you provide some instructions about that?

Comment (raulcabello) [reply README.md]: these file are inside the .zip file in the release, see example https://github.com/raulcabello/kwctl/releases/tag/v0.0.0.888
I explained it here https://github.com/kubewarden/kwctl/pull/230/commits/c57eacfae3dedff9f4b32f109930f086ea9b7fba
we should probably do a new kwctl release, so .sig and .pem files are available

File: README.md [hunk 2]
```diff
@@ -286,7 +286,8 @@ Then start a new shell or run `source ~/.zshrc` once.
 ## Verify kwctl binaries
 
 kwctl binaries are signed using [Sigstore](https://docs.sigstore.dev/cosign/working_with_blobs/#signing-blobs-as-files). 
-Each release contains two files: kwctl.sig and kwctl.pem that can be used for verification.
+When you download a [kwctl release](https://github.com/kubewarden/kwctl/releases/) each zip file contains two 
+files that can be used for verification: kwctl.sig and kwctl.pem.
```
Comment (flavio) [inline README.md:290]: nit:
```suggestion
files that can be used for verification: `kwctl.sig` and `kwctl.pem`.
```

**Review by flavio — APPROVED** [review]
I left a small comment. Feel free to merge this PR once they get accepted

**Pull request merged**